src/Microsoft.ApplicationInsights/Managed/Shared/Extensibility/Implementation/Tracing/HealthHeartbeatDefaultPayload.cs

```diff
@@ -0,0 +1,82 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.Implementation.Tracing
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    internal class HealthHeartbeatDefaultPayload : IHealthHeartbeatPayloadExtension
+    {
+        public const string FieldRuntimeFrameworkVer = "runtimeFramework";
+        public const string FieldTargetFramework = "targetFramework";
+        public const string FieldAppInsightsSdkVer = "appinsightsSdkVer";
+
+        public string EnabledProperties;
```

Comment:
A public, writable field is unusual and usually a poor design choice - even for an internal class. Either make it `readonly` or turn it into an auto-property.

Reply:
Thanks for pointing this out - was concentrating on unit tests at the time and not design principles! Will take extra care in subsequent PR updates...